src/include/gurt/common.h

```diff
@@ -156,7 +156,7 @@ extern "C" {
 		int _esz = (int)(size);					\
 		int _sz = (int)(size) * (cnt);				\
 		/* Compiler check to ensure type match */		\
-		typeof(newptr) optr = oldptr;				\
+		__typeof__(newptr) optr = oldptr;				\
```

Comment:
I think just removing a tab will silence the checkpatch warning

Reply:
I will do that. But the checkpatch exposes a whole lot of other legacy warnings after this